src/systems/variational_smoother_system.C

```diff
@@ -195,7 +195,7 @@ void VariationalSmootherSystem::prepare_for_smoothing()
                   // Scale the nodal positions to conserve area
                   auto node_ptr = std::make_unique<Node>(
                       equilateral_points[node_id] * side_length, node_id);
-                  target_elem->set_node(node_id) = node_ptr.get();
+                  target_elem->set_node(node_id, node_ptr.get());
```

Comment:
I think I'll make a separate PR with just this fix as well, it shouldn't conflict with #4218 and it will give us working `--disable-deprecated` build before #4218 just in case that is useful for debugging for some reason...